search/query.go

```diff
@@ -25,12 +25,22 @@ const (
 	NUM_SUGGESTS = 500
 )
 
-func createResultsQuery(resultTypes []string, q Query) elastic.Query {
+func createResultsQuery(resultTypes []string, q Query, docIds []string) elastic.Query {
 	boolQuery := elastic.NewBoolQuery().Must(
 		elastic.NewConstantScoreQuery(
 			elastic.NewTermsQuery("result_type", utils.ConvertArgsString(resultTypes)...),
 		).Boost(0.0),
+		//elastic.NewConstantScoreQuery(
```

Comment:
Why do you need commented out code here?

Reply:
removed